system/datatypes/peer_reviewed_article.php

```diff
@@ -1793,22 +1793,11 @@ protected function getStatus() {
 	 * @return bool
 	 */
 	protected function canBeSetAsApproved() {
-		/** @var HyphaDomElement $discussions */
-		$discussions = $this->xml->find(self::FIELD_NAME_DISCUSSION_CONTAINER);
-		/** @var NodeList $blockingDiscussions */
-		$xpath = './/' . self::FIELD_NAME_DISCUSSION . '[@' . self::FIELD_NAME_DISCUSSION_BLOCKING . '="1" and @' . self::FIELD_NAME_DISCUSSION_BLOCK_RESOLVED . '!="1"]';
-		$blockingDiscussions = $discussions->findXPath($xpath);
-		$blockingDiscussionsCount = $blockingDiscussions->count();
-		if ($blockingDiscussionsCount > 0) {
-			return false;
-		}
-
-		$approveCount = $this->xml->find(self::FIELD_NAME_APPROVE_CONTAINER)->children()->count();
-		if ($approveCount < self::APPROVES_NEEDED) {
-			return false;
-		}
+		list($_checkStatus, $checks) = $this->runChecks();
```

Comment:
didn't know a better place to put this comment.. There is a typo in the commit message

Reply:
Found it, fixed :-)